src/RoomArea/ProjectRoom.js

```diff
@@ -129,52 +129,47 @@ const ProjectRoom = ({checked}) => {
+                                        )}
+                                        </>
+                                    ))} 
                                         
-                                        <h3> Advisors </h3>
-                                            {roomInfo && roomInfo.map((member)=> ( 
-                                                <>
-                                                {member.user_role === "Advisor" && (
-                                                <li>{member.first_name} {member.last_name}</li>
-                                                )}
-                                                </>
-                                            ))} 
-                                        </div>
-                                    <h2>Abstract</h2>
-                                     <p>{nArr[2]}</p>                                  
-                                </div>
-                            )}
-
-
-                            <div className="bbb">
-                                {/* Update to get src url from the backend. Temporarily Hardcoded to get a view working  */}
-                                {/* <iframe className="temp" src="https://iapstream.ece.uprm.edu/bigbluebutton/api/create?name=DemoMeeting&meetingID=DemoMeeting&attendeePW=ap&moderatorPW=mp&checksum=f5e85d6b55189f228cf06e4791736e44b63282f1"></iframe>  */}
-                                <br></br>
-                                <iframe className="iframe" src={bbbUrl} allow="camera;microphone;display-capture" allowFullScreen></iframe>  
+                                    <h3> Advisors </h3>
+                                    {roomInfo && roomInfo.map((member)=> ( 
+                                        <>
+                                        {member.user_role === "Advisor" && (
+                                        <li>{member.first_name} {member.last_name}</li>
+                                        )}
+                                        </>
+                                    ))} 
+                                    </div>
+                                <h2>Abstract</h2>
+                                <p>{nArr[2]}</p>                                  
                             </div>
-                            
-                        </div>
-
+                        )}
 
+                        <div className="bbb">
+                            {/* Update to get src url from the backend. Temporarily Hardcoded to get a view working  */}
+                            {/* <iframe className="temp" src="https://iapstream.ece.uprm.edu/bigbluebutton/api/create?name=DemoMeeting&meetingID=DemoMeeting&attendeePW=ap&moderatorPW=mp&checksum=f5e85d6b55189f228cf06e4791736e44b63282f1"></iframe>  */}
+                            <br></br>
+                            <iframe className="iframe" src={bbbUrl} allow="camera;microphone;display-capture" allowFullScreen></iframe>  
```

Comment:
the Frontend is unable to resolve the bbbUrl src parameter at this time.

The behavior causes the iframe to load empty at this time.